forte/data/readers/multiple_choice_qa_reader.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+The reader that reads multiple choice qa data into Datapacks.
+"""
+import os
+import json
+from typing import Iterator, Any, List
+
+from forte.data.data_pack import DataPack
+from forte.data.io_utils import dataset_path_iterator
+from forte.data.readers.base_reader import PackReader
+from ft.onto.mutliple_choice_qa_ontology import Article, Passage, \
+    Question, Option
+
+__all__ = [
+    "MultipleChoiceQAReader",
+]
+
+
+class MultipleChoiceQAReader(PackReader):
+    """
+    :class:`MultipleChoiceQAReader` is designed to read in multiple choice qa
+    dataset.
+    """
+
+    # pylint: disable=no-self-use
+    def _collect(self, json_directory) -> Iterator[Any]:  # type: ignore
+        """
+        Should be called with param `json_directory` which is a path to a folder
+        containing json files.
+        Args:
+            json_directory: directory containing the json files.
+
+        Returns: Iterator over paths to .json files
+
+        """
+        return dataset_path_iterator(json_directory, "")
```

Comment:
I wanted to allow both .json and .txt extension here but we can pass only a single extension to this function.
Should I modify it to allow passing a list of extensions?